example/tsconfig.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "@tsconfig/react-native/tsconfig.json"
```

Comment:
doesn't this make the example app use a different tsconfig? and make it so that running `tsc` won't catch errors in `example` app?